src/components/review/WriteReview.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import axiosInstance from '../../apis/axiosClient';
+import { useAuth } from '@akosasante/react-auth-context';
+const WriteReview = ({ albumId, refreshReviews }) => {
+  const [formData, setFormData] = useState({
+    user: null,
+    album: null,
+    title: '',
+    rating: '',
+    comment: '',
+  });
+  const [submitting, setSubmitting] = useState(false);
+  const [error, setError] = useState(null);
+  const { user } = useAuth();
```

Comment:
console log this value; I think the actual user object you want is a little nested in there. This is something we can fix/simplify in some future work; but for now, you need to do `user.user.<whatever field you want>` to access things properly